pynestml/frontend/pynestml_frontend.py

```diff
@@ -117,12 +119,32 @@ def install_nest(target_path: str, nest_path: str) -> None:
     nest_path : str
         Path to the NEST installation, which should point to the main directory where NEST is installed. This folder contains the ``bin``, ``lib(64)``, ``include``, and ``share`` folders of the NEST install. The ``bin`` folder should contain the ``nest-config`` script, which is accessed by NESTML to perform the installation. This path is the same as that passed through the ``-Dwith-nest`` argument of the CMake command before building the generated NEST module. The suffix ``bin/nest-config`` will be automatically appended to ``nest_path``.
 
+    install_path: str
+        Path to the install directory, where the generated module library will be created.
+
     Raises
     ------
     GeneratedCodeBuildException
         If any kind of failure occurs during cmake configuration, build, or install.
     '''
-    nest_installer(target_path, nest_path)
+    if install_path is not None:
+        nest_installer(target_path, nest_path, install_path)
+        # add the install_path to the python library
+        system = platform.system()
+        lib_key = ""
+
+        if system == "Linux":
+            lib_key = "LD_LIBRARY_PATH"
+        else:
+            lib_key = "DYLD_LIBRARY_PATH"
+
+        lib_path = os.path.join(install_path, "lib", "nest")
+        if lib_key in os.environ:
+            os.environ[lib_key] += os.pathsep + lib_path
```

Comment:
might be good to check if the path is already in the environment variable and warn the user that it should be added there if not, especially since users that are going to encounter this issue are likely to be ones that did not compile NEST themselves and may not be familiar with the notion of paths